wgsl/index.bs

```diff
@@ -1218,58 +1228,68 @@ Each storage class has unique properties determining
 mutability, visibility, the values it may contain,
 and how to use variables with it.
 
-<table class='data'>
+<table class='data' id="storage-class-table">
   <caption>Storage Classes</caption>
   <thead>
     <tr><th>Storage class
-        <th>Readable by shader?<br>Writable by shader?
         <th>Sharing among invocations
+        <th>Supported access modes
+        <th>Default access mode
```

Comment:
I find it slightly confusing that we specify both the supported and the default, even though we don't support changing this. How do you feel about leaving the Default blank for classes that don't support changing it?

Reply:
I can remove the column and annotate the 'read' case as default in the row for 'storage'.
I'll do that.

Reply:
This is the new version:   ![Screen Shot 2021-05-19 at 4 52 05 PM](https://user-images.githubusercontent.com/13054574/118882824-951ffd80-b8c2-11eb-9a66-32091cb0f1e7.png)